### PR TITLE
Add database.

### PR DESCRIPTION
**Pull request by mhbratek**
Add persistence to the app. Hibernate configuration.
Update pom.xml and gitignore.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -33,40 +84,254 @@
...
+                    <reportSet>
+                        <reports>
+                            <report>report</report>
+                        </reports>
+                    </reportSet>
+                </reportSets>
+            </plugin>
+        </plugins>
+    </reporting>
+
```
Comment (patkkka) [inline pom.xml:330]: nice comments improving readability :)

**Review by patkkka — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -58,7 +58,7 @@ test-output/
 
 # User-specific stuff:
 .idea/
-
+hibernate.properties
```
Comment (agawda) [inline .gitignore:61]: 👍

File: pom.xml [hunk 3]
```diff
@@ -6,24 +6,75 @@
...
+        <developer>
+            <name>Pawel Stabryla</name>
+            <email>Pawel_Stabryla@epam.com</email>
+        </developer>
+    </developers>
 
 	<properties>
-		<!-- JAVA & ENCODING -->
+
+        <!-- JAVA & ENCODING -->
```
Comment (agawda) [inline pom.xml:32]: 👍  for sections.

File: pom.xml [hunk 4]
```diff
@@ -6,24 +6,75 @@
...
+		<!-- PERSISTENCE -->
+		<hibernate.version>5.1.0.Final</hibernate.version>
+		<spring.data.version>1.10.2.RELEASE</spring.data.version>
+		<mysql.connector.version>5.1.38</mysql.connector.version>
+		<hibernate.validator.version>4.3.1.Final</hibernate.validator.version>
+
+		<!-- Utils -->
+		<slf4j.version>1.7.5</slf4j.version>
 		
 		<!-- PLUGINS -->
```
Comment (agawda) [inline pom.xml:50]: I'd write `MAVEN PLUGINS` here, but maybe it's just me. :P

File: src/main/java/com/java/academy/configuration/HibernateConfiguration.java [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.sql.DataSource;
+import java.util.Properties;
+
+@Configuration
+@PropertySource(value = {"classpath:hibernate.properties"})
+@EnableJpaRepositories(basePackages = "com.java.academy.dao")
+public class HibernateConfiguration {
```
Comment (agawda) [inline src/main/java/com/java/academy/configuration/HibernateConfiguration.java:23]: You have a lot of unnecessary empty lines at the end of the file.

**Review by agawda — APPROVED** [review]

**Pull request merged**